sw/host/opentitanlib/src/tpm/status.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::fmt;
+
+pub struct TpmStatus(u32);
```

Comment:
Would this struct be better modeled with the `bitflags!` macro?

Reply:
burst_count is a 16 bit field, so this one may not be better as a `bitflags!`.